nt-discovery.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+run_command() {
+	# $1: command to run
```

Comment:
I think we usually use 2 spaces for indentation in shell scripts, in both this and other repos. See for example `tests/aws-spawn-test.sh`.